.github/BOTMETA.yml

```diff
@@ -24,7 +23,6 @@ files:
     - esx
     - vcenter
     - vcsa
-    - vcsim
```

Comment:
The changes in `.github/` are a source of problem. The Github API prevents the merge of a PR if it touch this directory and the PR is not rebased on the head. I would suggest to revert these two lines for now.